A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -316,6 +316,10 @@ class FMCMainDisplay extends BaseAirliners {
         this._inOutElement.style.color = color;
     }
 
+    isLockedInit() {
+        return Simplane.getEngineActive(0) || Simplane.getEngineActive(1);
+    }
```

Comment:
Perhaps rename this method to something more descriptive. I'm sure this method can be called in many other places. perhaps something like `isAnEngineOn()`. Also add jsDoc so contirbutor's IDEs can make use of it.

Reply:
already did, will commit when tested changes